corehq/apps/change_feed/management/commands/validate_kafka_pillow_checkpoints.py

```diff
@@ -30,7 +31,8 @@ def handle(self, **options):
 def validate_checkpoints(print_only):
 
     for pillow in get_all_pillow_instances():
-        if isinstance(pillow.get_change_feed(), KafkaChangeFeed):
+        if (pillow.pillow_id in getattr(settings, 'ACTIVE_PILLOW_NAMES', []) and
```

Comment:
if `ACTIVE_PILLOW_NAMES` is not in settings then it should check them all (to remain consistent with the current functionality)

Reply:
That's right, updated.